Array_Vector_Exercises/Exercise1/TDinamicArray.h

```diff
@@ -0,0 +1,242 @@
+#pragma once
+
+template<typename T>
+class TDynamicArray
+{
+private:
+	int SizeArray;
```

Comment:
Remember that it's a bit redundant to call all your variables Array. Size and Capacity would be more than enough since their context is very clear. This reduces noise in the implementations.